docs/continuous_integration.md

```diff
@@ -1,16 +1,34 @@
 # vHive CI guide
 
 ## Deploying GitHub self-hosted runners
-* Create Create GitHub personal access token for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
+
+### Deploying integration tests workflow runners
+1. Create GitHub personal access token for your repo with [these](https://github.com/myoung34/docker-github-actions-runner#create-github-personal-access-token) scopes.
+
+2. Clone the repository.
+
+```bash
+git clone https://github.com/vhive-serverless/vhive && cd vhive
+```
+
+3. Setup the desired number of runners:
+
+```bash
+./scripts/github_runner/setup_integ_runners.sh <num of runners> <GitHub Account> <Github PAT> [restart]
```

Comment:
please specify `Github account` format and clarify what `restart` means

Reply:
I don't know how to specify GitHub account... I believe it's self-explanatory. Like `CuriousGeorgiy` or `ustiugov` or `vhive-serverless`.

Added a comment about `restart` flag.

Reply:
I don't see the comment about restart, where?

just give these examples: vhive-serverless or your-username

Reply:
I pushed the wrong branch, should be up to date now. Added example.